Makefile

```diff
@@ -10,6 +10,20 @@ deps:
 	# Update dependencies
 	git submodule update --init --recursive
 
+.PHONY: install
+install: deps
+	pip install -e .[cpu,tests,docs]
+	pip install pycocotools==2.0.0
+
+.PHONY: install-gpu
+install-gpu: install
+	pip install -e .[gpu]
+	pip install -e .[dist]
+
+.PHONY: lint
+lint:
+	flake8 ./blueoil --exclude=templates,converter
```

Comment:
Isn't `test-blueoil-pep8` enough?

Reply:
1st, this is for development.
2nd, for flake8, using docker sounds too complecated

Reply:
I see.